MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -1072,11 +1073,31 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         gridBagConstraints.anchor = java.awt.GridBagConstraints.WEST;
         panSubAcquire.add(choiceAcquireSkill, gridBagConstraints);
 
+        DefaultComboBoxModel<String> acquisitionSortPriorityModel = new DefaultComboBoxModel<String>();
+        acquisitionSortPriorityModel.addElement(CampaignOptions.S_BESTFIRST);
+        acquisitionSortPriorityModel.addElement(CampaignOptions.S_WORSTFIRST);
+        acquisitionSortPriorityModel.setSelectedItem(options.getAcquisitionSortPriority());
+        choiceAcquireSortPriority = new JComboBox<String>(acquisitionSortPriorityModel);
+
+        gridBagConstraints = new java.awt.GridBagConstraints();
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 3;
+        gridBagConstraints.fill = java.awt.GridBagConstraints.NONE;
+        gridBagConstraints.anchor = java.awt.GridBagConstraints.WEST;
+        panSubAcquire.add(new JLabel("Buyer Skill Priority:"), gridBagConstraints);
```

Comment:
I'd like to see new text turned into text resources instead of being hard-coded. 

Ideally, the other ones too, but let's not get too carried away.